cudapoa/include/claraparabricks/genomeworks/cudapoa/cudapoa.hpp

```diff
@@ -37,6 +37,7 @@ enum StatusType
     exceeded_maximum_sequences_per_poa,
     node_count_exceeded_maximum_graph_size,
     edge_count_exceeded_maximum_graph_size,
+    exceeded_adaptive_banded_matrix_size,
```

Comment:
previously we have removed a redundant error code, so the order historically has been changed. How about printing an error code description, I guess currently we do it only in debug mode. I don't have problem with moving it to the end, but it will look a bit out of place.

Reply:
Fair enough, okay to leave as is then. And I agree, an error description function is longtime pending...